services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/unit_templates_manager/unit_template_second_row.jsx

```diff
@@ -27,15 +27,15 @@ export default class UnitTemplateSecondRow extends React.Component {
     return [data.time, 'mins'].join(' ');
   };
 
-  renderReadability = () => {
+  renderGradeLevelRange = () => {
     const { data, } = this.props
 
-    if (!data.readability) { return <span />}
+    if (!data.grade_level_range) { return <span />}
 
     return (
-      <div className="readability">
+      <div className="grade-level-range">
```

Comment:
Just wanted to flag how much I appreciate the bit of extra work it takes to rename classes like this in order to keep things consistent.  Thanks!